examples/post/index.html

```diff
@@ -24,15 +24,24 @@ <h1>axios.post</h1>
         document.getElementById('post').onclick = function () {
           var data = document.getElementById('data').value;
 
-          axios.post('/post/server', JSON.parse(data))
-            .then(function (res) {
-              output.className = 'container';
-              output.innerHTML = res.data;
-            })
-            .catch(function (err) {
-              output.className = 'container text-danger';
-              output.innerHTML = err.message;
-            });
+          try {
+              data= JSON.parse(data);
+          } catch (e) {
+              output.innerHTML = "Error : Is blank or not JSON string type. Please check your data.";
+              data = null;
+          }
+
+          if (data != null) {
+              axios.post('/post/server', data)
+              .then(function (res) {
+                  output.className = 'container';
+                  output.innerHTML = res.data;
+              })
+              .catch(function (err) {
+                  output.className = 'container text-danger';
+                  output.innerHTML = err.message;
+              });
```

Comment:
And I'd like move all these thing into the try block. Please keep the same style as well as the whole file.